API_DESIGN_GUIDE.md

```diff
@@ -41,12 +41,13 @@ try:
 except ImportError:
     pass
```

Comment:
See comment in issue, we should just do rouge_score = None to avoid needing to mess with `sys.modules`.